ros2action/ros2action/api/__init__.py

```diff
@@ -139,3 +142,27 @@ def __call__(self, prefix, parsed_args, **kwargs):
                 if n == action_name:
                     return t
         return []
+
+
+class ActionGoalPrototypeCompleter:
+    """Callable returning an action goal prototype."""
+
+    def __init__(self, *, action_type_key=None):
+        self.action_type_key = action_type_key
+
+    def __call__(self, prefix, parsed_args, **kwargs):
+        full_action_type = getattr(parsed_args, self.action_type_key)
+        try:
+            parts = full_action_type.split('/')
+            if len(parts) == 1:
+                raise ValueError()
+            if 'action' not in parts:
```

Comment:
I think this is not strictly correct.
I'm adding an utilities module to `rosidl_runtime_py` for this (https://github.com/ros2/rosidl_python/pull/78/files).
You can wait until it gets merged, or do a similar logic here (in that case, I will later use the new methods through this repo in a followup PR).

Reply:
A quick try seemed not to work if `action` was not specified in the full name, thus this check/insert.

I'll rather wait for https://github.com/ros2/rosidl_python/pull/78 to be merged, it seems small enough to move fast.

Reply:
maybe the following would be better (?)
```python
if parts[-1] != 'action':
```

Reply:
@ivanpauno I updated this PR using the utilities from https://github.com/ros2/rosidl_python/pull/78